[2025-01-14T05:34:04.376+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL_influencer_marchendise.data_extract manual__2025-01-14T05:34:03.615809+00:00 [queued]>
[2025-01-14T05:34:04.381+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL_influencer_marchendise.data_extract manual__2025-01-14T05:34:03.615809+00:00 [queued]>
[2025-01-14T05:34:04.382+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-01-14T05:34:04.382+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-01-14T05:34:04.382+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-01-14T05:34:04.388+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): data_extract> on 2025-01-14 05:34:03.615809+00:00
[2025-01-14T05:34:04.391+0000] {standard_task_runner.py:52} INFO - Started process 211 to run task
[2025-01-14T05:34:04.393+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_influencer_marchendise', 'data_extract', 'manual__2025-01-14T05:34:03.615809+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/AirflowM3.py', '--cfg-path', '/tmp/tmpbfzbhajs', '--error-file', '/tmp/tmptxj9s2th']
[2025-01-14T05:34:04.394+0000] {standard_task_runner.py:80} INFO - Job 2: Subtask data_extract
[2025-01-14T05:34:04.428+0000] {task_command.py:371} INFO - Running <TaskInstance: ETL_influencer_marchendise.data_extract manual__2025-01-14T05:34:03.615809+00:00 [running]> on host e262d4edf520
[2025-01-14T05:34:04.463+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=afif
AIRFLOW_CTX_DAG_ID=ETL_influencer_marchendise
AIRFLOW_CTX_TASK_ID=data_extract
AIRFLOW_CTX_EXECUTION_DATE=2025-01-14T05:34:03.615809+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-01-14T05:34:03.615809+00:00
[2025-01-14T05:34:04.464+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-01-14T05:34:04.465+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/dags/extract.py']
[2025-01-14T05:34:04.470+0000] {subprocess.py:85} INFO - Output:
[2025-01-14T05:34:05.341+0000] {subprocess.py:92} INFO - WARNING: An illegal reflective access operation has occurred
[2025-01-14T05:34:05.342+0000] {subprocess.py:92} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.2.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2025-01-14T05:34:05.343+0000] {subprocess.py:92} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2025-01-14T05:34:05.343+0000] {subprocess.py:92} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2025-01-14T05:34:05.343+0000] {subprocess.py:92} INFO - WARNING: All illegal access operations will be denied in a future release
[2025-01-14T05:34:05.538+0000] {subprocess.py:92} INFO - 25/01/14 05:34:05 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-01-14T05:34:05.615+0000] {subprocess.py:92} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2025-01-14T05:34:05.616+0000] {subprocess.py:92} INFO - Setting default log level to "WARN".
[2025-01-14T05:34:05.616+0000] {subprocess.py:92} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-01-14T05:34:10.034+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-01-14T05:34:10.092+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=ETL_influencer_marchendise, task_id=data_extract, execution_date=20250114T053403, start_date=20250114T053404, end_date=20250114T053410
[2025-01-14T05:34:10.113+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-01-14T05:34:10.130+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
